benchmarks/src/bin/tpch.rs

```diff
@@ -562,7 +568,10 @@ mod tests {
     }
 
     fn create_context() -> Result<SessionContext> {
```

Comment:
I fix the parameter `target_partitions` so that the generated plans are the same in different machines.